docs/how-tos/provision.md

```diff
@@ -0,0 +1,310 @@
+
+{{% /expand%}}
+
+{{% expand "A machine fragment" %}}
+
+Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ for copying parts of a configuration file to multiple machines.
+
+Follow the steps to [Create a configuration fragment](/how-tos/one-to-many/) and make a note of the fragment ID.
+
+{{% /expand%}}
+
+## Decide on the provisioning method
+
+You can choose to let your end users complete machine setup by using a captive web portal or a mobile app.
+
+If you choose to have a mobile app experience, you can use the [Viam mobile app](/fleet/control/#control-interface-in-the-viam-mobile-app) or create your own custom mobile app using the [Flutter SDK](https://flutter.viam.dev/viam_protos.provisioning.provisioning/ProvisioningServiceClient-class.html) or the [TypeScript SDK](https://github.com/viamrobotics/viam-typescript-sdk/blob/main/src/app/provisioning-client.ts) to connect to `viam-agent` and provision your machines.
+
+If you choose to use the Viam mobile app, you must provide a fragment for provisioning.
+
+If you choose to use the captive web portal, you can optionally create a machine already and provide its credentials configuration file at <FILE>/etc/viam.json</FILE>.
+
+You can get the `viam-server` app JSON configuration by clicking the copy icon next to **Viam server config** in the part status dropdown to the right of your machine's name on the top of the page.
+
+{{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown">}}
+
+## Configure `agent-provisioning`
+
+{{< table >}}
+
+{{% tablestep link="/fleet/provision/#configuration" %}}
+**Configure provisioning**
+
+You may optionally provide a provisioning configuration file to customize the experience.
+Place the file at <file>/etc/viam-provisioning.json</file> with the following format:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "manufacturer": "<NAME>", # your company name
+  "model": "<NAME>", # the machine's model
+  "fragment_id": "<ID>", # the fragment id, required for mobile app
+  "hotspot_prefix": "<PREFIX>", # machine creates a hotspot during setup
+  "disable_dns_redirect": true, # disable if using a mobile app
+  "hotspot_password": "<PASSWORD>", # password for the hotspot
+}
+```
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Install `viam-agent`
+
+`viam-agent` is a self-updating service manager that maintains the lifecycle for several Viam services and keeps them updated.
+If you intend to use `viam-agent` to keep your device's Viam software up-to-date or to use its provisioning plugin to let end users set up their own machines, you need to use `viam-agent` and install it on the machine before sending it to the user.
+
+You can [install `viam-agent` on live systems](/installation/) or preinstall it:
+
+{{< tabs >}}
+{{% tab name="Preinstall on an image" %}}
```

Comment:
These instructions are (partially) duplicating the SD card tab. The preinstall script can install to a mounted FS image (that's typically an SD card) OR it can create a tarball (which is the second half of these instructions.)